src/runtime/metal/metal_device_api.mm

```diff
@@ -183,16 +187,25 @@ int GetWarpSize(id<MTLDevice> dev) {
   }
 }
 
+Stream* getStream(TVMStreamHandle stream, int device_id) {
```

Comment:
I think we prefer `GetStream`? I don't know the convention of Metal though

Reply:
Done